### PR TITLE
[DOCS] Adds Transform alerts docs

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR adds documentation to the Transform alert function and its transform health rule type.

### Preview

[Transform rules](https://elasticsearch_78185.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/transform-alerts.html)

### CODE REVIEW

File: docs/reference/transform/transform-alerts.asciidoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+The following {transform} rules are available:
+
+{transform-cap} health:: 
+  Monitors {transforms} health and alerts if an operational issue occurred.
+
+
+[[creating-transform-rules]]
+== Creating a rule
+
+You can create {transform} rules under **{stack-manage-app} > {alerts-ui}**.
```
Comment (darnautov) [inline docs/reference/transform/transform-alerts.asciidoc:24]: I see "Alerts and Actions" in the preview but it should be "Rules and Connectors"

Comment (darnautov) [reply docs/reference/transform/transform-alerts.asciidoc:24]: I noticed a similar issue with the last paragraph about defining actions

Comment (szabosteve) [reply docs/reference/transform/transform-alerts.asciidoc:24]:
```suggestion
You can create {transform} rules under **{stack-manage-app} > {rules-ui}**.
```

File: docs/reference/transform/transform-alerts.asciidoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+image::images/transform-alert-actions.png["Selecting connector type"]
+
+
+For example, you can choose _Slack_ as a connector type and configure it to send 
+a message to a channel you selected. You can also create an index connector that 
+writes the JSON object you configure to a specific index. It's also possible to 
+customize the notification messages. A list of variables is available to include 
+in the message, like {transform} ID, description, {transform} state, and so on.
+
+After you save the configurations, the rule appears in the *{alerts-ui}* list
```
Comment (szabosteve) [inline docs/reference/transform/transform-alerts.asciidoc:80]:
```suggestion
After you save the configurations, the rule appears in the *{rules-ui}* list 
```

File: docs/reference/transform/transform-alerts.asciidoc (hunk 1 above)
Comment (szabosteve) [reply docs/reference/transform/transform-alerts.asciidoc]: Addressed in aa0b52d.

File: docs/reference/transform/transform-alerts.asciidoc [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+in the message, like {transform} ID, description, {transform} state, and so on.
+
+After you save the configurations, the rule appears in the *{rules-ui}* list 
+where you can check its status and see the overview of its configuration 
+information.
+
+The name of an alert is always the same as the {transform} ID of the associated 
+{transform} that triggered it. You can mute the notifications for a particular 
+{transform} on the page of the rule that lists the individual alerts. You can 
+open it via *{alerts-ui}* by selecting the rule name.
```
Comment (darnautov) [inline docs/reference/transform/transform-alerts.asciidoc:87]:
```suggestion
open it via *{rules-ui}* by selecting the rule name.
```

**Review by darnautov — APPROVED** [review]

**Pull request merged**